packages/shared/src/lib/featureManagement.ts

```diff
@@ -43,6 +44,7 @@ const feature = {
   searchSources: new Feature('search_sources', false),
   onboardingChecklist: new Feature('onboarding_checklist', false),
   feedLayoutPreview: new Feature('feed_layout_preview', false),
+  onboardingBanner: new Feature('onboarding_banner', OnboardingBanner.Control),
```

Comment:
@rebelchris could you create this in GB?

Reply:
done